scripts/launch/connect-to-cluster/default.nix

```diff
@@ -82,5 +83,7 @@ in pkgs.writeScript "${executable}-connect-to-${environment}" ''
     --db-path "${stateDir}/db"                                     \
     ${ ifWallet "--wallet-db-path '${stateDir}/wallet-db'"}        \
     --keyfile ${stateDir}/secret.key                               \
-    ${ ifWallet "--wallet-address ${walletListen}"}
+        ${ ifWallet "--wallet-address ${walletListen}"}    \
+        --ekg-server 127.0.0.1:8080 --metrics +RTS -T -RTS             
```

Comment:
docker cant forward things to `127.0.0.1`, thats why the wallet has to listen on `0.0.0.0` when docker is in use